.gitignore

```diff
@@ -1,3 +1,6 @@
+# Ignore IDE files
+.idea
```

Comment:
You should rather use `git exclude` (https://help.github.com/articles/ignoring-files/#explicit-repository-excludes) to ignore it locally than ignoring it. But that's just nutpicking. ^^